aya/src/maps/xdp/xsk_map.rs

```diff
@@ -78,4 +78,23 @@ impl<T: BorrowMut<MapData>> XskMap<T> {
         )?;
         Ok(())
     }
+
+    /// Removes the `AF_XDP` socket stored at `index` from the map.
+    ///
+    /// Returns [`MapError::OutOfBounds`] if `index` is out of bounds, [`MapError::SyscallError`]
+    /// if `bpf_map_update_elem` fails.
+    pub fn clear_index(&mut self, index: u32) -> Result<(), MapError> {
```

Comment:
This should likely be either `remove` or `delete`. Hold please while we hear from @alessandrod.